R/content.R

```diff
@@ -32,7 +32,7 @@ Content <- R6::R6Class(
     },
     #' @description Obtain the content data from the Connect server.
     get_content_remote = function() {
-      new_content_details <- self$get_connect()$content(self$get_content()$guid)
+      new_content_details <- self$get_connect()$content(self$get_content()$guid, include = NULL)
```

Comment:
Why `include = NULL`? I know we weren't including it before, but it's probably not bad to have it. Might let us do some other improvements like https://github.com/posit-dev/connectapi/issues/402.

Not saying you have to change this, but if you leave it as is, it might be nice to leave a comment/todo explaining that we don't _need_ to have this as `NULL`, just so we don't assume that it must be this way in the future.

Reply:
It was mainly to avoid unintentionally changing existing behavior (and also because changing it resulted in a bunch of test failures that I didn't fully understand, but now do thanks to @jonkeane's help). I've gone ahead and removed the `include = NULL` both here and at line 593, and updated the test cases accordingly. The unit tests now pass on my machine but seem to be failing on CI so I will need to investigate a little further.

Reply:
oh...one of the test files didn't get picked up by git as having been renamed so the new version was just hanging out as an untracked file on my machine. hopefully this should be fixed now.